### PR TITLE
Update scripts, SYCL code and readme, build w/ CUDA by default and change default compute capability

### PR DESCRIPTION
**Pull request by nmnobre**
Hi,

Just another slate of small fixes to complete my previous contribution.
The SYCL code is now ~~almost~~ untouched, straight out of DPCT ~~, with the exception of the type of the first parameter of `gpuAssert()`, which I had to manually change from `dpct::err0` to `int` since the former is not included in the version of the DPCT headers included in this repo. Once we merge #23 we'll be able to simply use the SYCL code straight out of the tool with no alterations~~. :)

Cheers,
-Nuno

### CODE REVIEW

**Comment by DuncanMcBain** [comment]
I've also merged #23 now too!

**Comment by nmnobre** [comment]
> I've also merged #23 now too!

Grand! I think it's looking good now, but do let me know otherwise. :)

File: src_sycl/simulator.dp.cpp [hunk 1]
```diff
@@ -341,15 +334,10 @@ namespace simulation {
...
-      the code.
-      */
       curr_pos += curr_vel * params.dt;
       pNextPos.x[id] = curr_pos.x;
       pNextPos.y[id] = curr_pos.y;
       pNextPos.z[id] = curr_pos.z;
    }
 
-}  // namespace simulation
+}  // namespace simulation
```
Comment (DuncanMcBain) [inline src_sycl/simulator.dp.cpp]:
```suggestion
}  // namespace simulation

```

File: src_sycl/simulator.dp.hpp [hunk 2]
```diff
@@ -167,4 +165,4 @@ namespace simulation {
       void recvFromDevice();
    };
 
-}  // namespace simulation
+}  // namespace simulation
```
Comment (DuncanMcBain) [inline src_sycl/simulator.dp.hpp]:
```suggestion
}  // namespace simulation

```

**Review by DuncanMcBain — COMMENTED** [review]
I think I have the suggestions correct for restoring some newlines on a couple of files!

File: src_sycl/simulator.dp.hpp (hunk 2 above)
Comment (nmnobre) [reply src_sycl/simulator.dp.hpp]: Ah! Sure! The reason these were removed is that DPCT just respects the lack of a new line at the end of the respective CUDA files (simulator.cu and simulator.cuh). I've added a new commit (added you as a co-author 🙂) which adds a new line at the end of all four files. This way, the SYCL files are still an untouched copy of the result of passing the new CUDA files (with the new line) through DPCT.

Comment (DuncanMcBain) [reply src_sycl/simulator.dp.hpp]: Perfect thanks!

**Comment by DuncanMcBain** [comment]
Perfect, thank you very much!

**Pull request merged**